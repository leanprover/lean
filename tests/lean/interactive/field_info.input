{"seq_num": 0, "command": "sync", "file_name": "f", "content": "def f (n:nat) := n^.to_string"}
{"seq_num": 1, "command": "info", "file_name": "f", "line": 1, "column": 20}
