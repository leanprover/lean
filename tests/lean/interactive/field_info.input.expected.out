{"message":"file invalidated","response":"ok","seq_num":0}
{"record":{"full-id":"nat.to_string","source":{"column":14,"file":"/library/init/data/to_string.lean","line":77},"type":"ℕ → string"},"response":"ok","seq_num":1}
